cookbooks/aws-parallelcluster-environment/spec/unit/recipes/login_nodes_keys_spec.rb

```diff
@@ -36,22 +42,23 @@
           runner = runner(platform: platform, version: version) do |node|
             node.override['cluster']['node_type'] = 'HeadNode'
             node.override['cluster']['scheduler'] = 'slurm'
-            node.override['cluster']['shared_dir_login_nodes'] = '/opt/parallelcluster/shared_login_nodes'
+            node.override['cluster']['shared_dir_login_nodes'] = SHARED_DIR_LOGIN_NODES
+            node.override['cluster']['cluster_config_version'] = CLUSTER_CONFIG_VERSION
           end
           runner.converge(described_recipe)
         end
         cached(:node) { chef_run.node }
 
         it 'creates the scripts directory' do
-          is_expected.to create_directory("#{node['cluster']['shared_dir_login_nodes']}/scripts").with(
+          is_expected.to create_directory("#{SHARED_DIR_LOGIN_NODES}/scripts").with(
             owner: 'root',
             group: 'root',
             mode:  '0744'
           )
         end
 
         it 'creates keys-manager.sh script' do
-          is_expected.to create_cookbook_file("#{node['cluster']['shared_dir_login_nodes']}/scripts/keys-manager.sh").with(
+          is_expected.to create_cookbook_file("/SHARED_DIR_LOGIN_NODES/scripts/keys-manager.sh").with(
```

Comment:
should this have #{} around it?

"/#{SHARED_DIR_LOGIN_NODES}"

Reply:
Good catch, addressing